lib/optimize/FlagIncludedChunksPlugin.js

```diff
@@ -9,25 +9,25 @@ class FlagIncludedChunksPlugin {
 	apply(compiler) {
 		compiler.hooks.compilation.tap("FlagIncludedChunksPlugin", (compilation) => {
 			compilation.hooks.optimizeChunkIds.tap("FlagIncludedChunksPlugin", (chunks) => {
-				chunks.forEach((chunkA) => {
-					chunks.forEach((chunkB) => {
+				for(const chunkA of chunks) {
+					loopB: for(const chunkB of chunks) {
```

Comment:
Label is not needed. A simple `continue` would be enough

Reply:
It is because of lines 25-27.

Reply:
ah I see